### PR TITLE
Auto regenerate LORA's fbs files

### PR DESCRIPTION
**Pull request by snnn**
### Description

A left-over of PR #22046

### CODE REVIEW

**Comment by snnn** [comment]
@luncliff , do you know why now we cannot download google benchmark's source code from vcpkg?

**Comment by snnn** [comment]
I think it is because of a regression in vcpkg-tool.  See: https://github.com/microsoft/vcpkg/issues/41199#issuecomment-2378255699 and https://github.com/microsoft/vcpkg-tool/pull/1501
We are using the 09-18 version of vcpkg-tool.

**Comment by luncliff** [comment]
> @luncliff , do you know why now we cannot download google benchmark's source code from vcpkg?

No. I didn't meet the message for several months. I will check the detail as soon as possible.

**Comment by snnn** [comment]
Recently vcpkg had a regression. Though the problem has been fixed, we are still using an unpatched one.

File: .github/workflows/windows.yml [hunk 1]
```diff
@@ -56,7 +56,6 @@ jobs:
         uses: lukka/run-vcpkg@v11
         with:
           vcpkgDirectory: "C:/vcpkg" # use VCPKG_INSTALLATION_ROOT of the image
-          doNotUpdateVcpkg: true
```
Comment (snnn) [inline .github/workflows/windows.yml:59]: I had to remove this line to get the latest vcpkg-tool.

File: .github/workflows/windows.yml [hunk 2]
```diff
@@ -56,7 +56,6 @@ jobs:
...
           runVcpkgInstall: true
```
Comment (luncliff) [inline .github/workflows/windows.yml:59]:
```suggestion
          runVcpkgInstall: true
          vcpkgGitCommitId: "c82f74667287d3dc386bce81e44964370c91a289" # 2024.09.30
```

**Comment by snnn** [comment]
Now it works. Thanks for the quick response.

**Review by luncliff — COMMENTED** [review]
Understood. For future reference, we can specify vcpkg commit like this.
https://github.com/microsoft/vcpkg/releases/tag/2024.09.30

File: .github/workflows/windows.yml (hunk 2 above)
Comment (snnn) [reply .github/workflows/windows.yml:59]: It seems this action can auto find vcpkg git commit id from vcpkg-configuration.json file.

**Review by edgchen1 — APPROVED** [review]

**Review by jchen351 — APPROVED** [review]

**Pull request merged**